tests/core.py

```diff
@@ -456,6 +457,40 @@ def trigga(context, obj):
             dag=self.dag)
         t.run(start_date=DEFAULT_DATE, end_date=DEFAULT_DATE, ignore_ti_state=True)
 
+        session = settings.Session()
+        new_dag_run = session.query(models.DagRun).filter(
+            models.DagRun.run_id == trigga.run_id).first()
+        self.assertEqual(new_dag_run.execution_date, DEFAULT_DATE)
+
+    def test_trigger_dagrun_order_modified(self):
+        """
+        Test TriggerDagRunOperator with changes in DagRunOrder
+        """
+        new_execution_date = datetime(2016, 1, 1)
+        new_dag_run_id = 'manual_run_id'
+        payload_key = 'message'
+        payload = {payload_key: 'Hello World'}
+
+        def trigga(context, obj):
+            obj.run_id = new_dag_run_id
+            obj.execution_date = new_execution_date
+            obj.payload = payload
+            if True:
```

Comment:
`if True` ?

Reply:
@anxodio are you still working on this PR?

Reply:
@anxodio  Seems like this was almost ready to merge so it's a waste to throw it away but if you are not going to work on it anymore can you close it? no point of having it pending any longer...

Reply:
@ron819 yes, I will close it. I think it is a good idea to implement something like this, but it's a breaking change and I think that must be accorded with the community before.